.github/workflows/build-windows.yml

```diff
@@ -177,7 +177,7 @@ jobs:
           ./bootstrap
           mkdir build
           cd build
-          cmake -G Ninja -DCMAKE_BUILD_TYPE=${{ steps.setup.outputs.build-type }} ..
+          cmake -G Ninja -DCMAKE_BUILD_TYPE=${{ steps.setup.outputs.build-type }} -DTRACY_ENABLE=ON -DTRACY_TIMER_FALLBACK=ON ..
```

Comment:
Could you include in this PR to not clear the ` target`  folder?
Also, I think we don't need to build libbacktrace anymore, do we?

Reply:
nope, didn't seem used at least